lib/src/enums.dart

```diff
@@ -0,0 +1,28 @@
+/// Values for the event name to be sent to Google Analytics
+///
+/// The [label] for each enum value is what will be logged, the [desc]
+/// is here for documentation purposes
+enum DashEvents {
+  hotReloadTime(
+    label: 'hot_reload_time',
+    desc: 'Hot reload duration',
+  ),
+  ;
+
+  final String label;
+  final String desc;
+  const DashEvents({
+    required this.label,
+    required this.desc,
+  });
+}
+
+/// Enumerate options for platform
+enum DevicePlatforms {
+  windows('Windows'),
+  macos('Macintosh'),
```

Comment:
I think this should be the name of the operating system:

```suggestion
  macos('macOS'),
```

Unless there is some other constraining factor?

Reply:
I'm indifferent to what we call them, I just asked JaYoung what they called them currently in GA3 and continued. I can relabel them though